moveit_ros/planning/planning_request_adapter_plugins/CMakeLists.txt

```diff
@@ -18,15 +20,3 @@ ament_target_dependencies(moveit_default_planning_request_adapter_plugins
   rclcpp
   pluginlib
 )
-
-add_executable(moveit_list_request_adapter_plugins src/list.cpp)
-ament_target_dependencies(moveit_list_request_adapter_plugins
-  Boost
-  moveit_core
-  rclcpp
-  pluginlib
-)
-
-install(TARGETS moveit_list_request_adapter_plugins
-  DESTINATION lib/${PROJECT_NAME}
-)
```

Comment:
I like the code cleanup here, and also agree that it's weird just having this as a standalone executable... but is there an alternative way to list planning adapters that we can still use?

I'm assuming yes, and it's a `ros2 param get` type of situation, right?

Reply:
No really, because this executable checks the available c++ plugins

Reply:
I moved the executable into its own directory so it is still usable + detectable. In a follow up PR this executable could be refactored to list planner_plugins or IK plugins as well :thinking: